cloud/stack/status_test.go

```diff
@@ -19,7 +19,7 @@ func TestStackStatus(t *testing.T) {
 
 	var s stack.Status
 	errtest.Assert(t, s.Validate(), errors.E(stack.ErrInvalidStatus))
-	s = stack.Canceled + 1
+	s = stack.Failed + 1
```

Comment:
Should we rather do this?
```suggestion
	s = stack.Unrecognized
```

Reply:
This is testing that status must be a power of two value, just in case someone does:
`s := Status(5)`.

Reply:
Oh right, I forgot that it's a bitmask, so I misinterpreted this line.